src/07-projects/projects.apib

```diff
@@ -211,6 +211,7 @@ Update a project.
                     + contact
                     + company
             + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string)
+        + budget (Money) - Null will reset the budget to zero
```

Comment:
Should we really allow passing null trough the API, since it's not technically nullable?

Reply:
Good point.